### ISSUE TITLE
OmniPath failling to install in GH action

### ISSUE THREAD

**Issue by joan-yanqiong** [saezlab/OmnipathR#101]
Hi, 

I'm using the `pkgdown` GitHub action, in my package I'm using OmniPathR, though it seems that it cannot be installed in the `r-lib/actions/setup-r-dependencies@v2` step of the GH action. Below providing part of the output of that step. 

For you information I added OmnipathR as follows in the DESCRIPTION file. 
```
Imports: 
OmnipathR
Remotes:
saezlab/OmnipathR, 

```

```
  ✔ Built sparseMatrixStats 1.16.0 (32.5s)
  ✔ Installed sparseMatrixStats 1.16.0  (96ms)
  ✔ Installed checkmate 2.3.1  (51ms)
  ✔ Installed logger 0.3.0  (53ms)
  ℹ Packaging OmnipathR 3.13.5
  ✔ Packaged OmnipathR 3.13.5 (1.9s)
  ℹ Building OmnipathR 3.13.5
  ✖ Failed to build OmnipathR 3.13.5 (1.2s)
  Error: 
  ! error in pak subprocess
  Caused by error in `stop_task_build(state, worker)`:
  ! Failed to build source package OmnipathR.
  ---
  Backtrace:
  1. pak::lockfile_install(".github/pkg.lock")
  2. pak:::remote(function(...) { …
  3. err$throw(res$error)
  ---
  Subprocess backtrace:
   1. base::withCallingHandlers(cli_message = function(msg) { …
   2. get("lockfile_install_internal", asNamespace("pak"))(...)
   3. plan$install()
   4. pkgdepends::install_package_plan(plan, lib = private$library, num_workers = nw, …
   5. base::withCallingHandlers({ …
   6. pkgdepends:::handle_events(state, events)
   7. pkgdepends:::handle_event(state, i)
   8. pkgdepends:::stop_task(state, worker)
   9. pkgdepends:::stop_task_build(state, worker)
  10. base::throw(pkg_error("Failed to build source package {.pkg {pkg}}.", …
  11. | base::signalCondition(cond)
  12. global (function (e) …
  Execution halted
  Error: Process completed with exit code 1.
```

Please let me know if I can provide any other information.

**Comment by enblacar** [comment]
Bumping this up. 

Would it be possible to get an estimate of when this would be fixed? Many thanks!

Best,
Enrique

**Comment by deeenes** [comment]
I'm closing this one too, has been fixed, see #94
